src/Recruiter/Worker.php

```diff
@@ -126,10 +126,11 @@ private function afterExecutionOf($job)
             $this->memoryLimit->ensure(memory_get_usage());
         } catch (MemoryLimitExceededException $e) {
             printf(
```

Comment:
this should be fixed using the Logger
Not so simple because Worker does not contains Logger instance and it is created by Recruiter which does not contains Logger instance also